infrasim/ipmiconsole/__init__.py

```diff
@@ -152,9 +152,28 @@ def start(instance="default"):
     sdr.parse_sdrs()
     # running thread for each threshold based sensor
     _spawn_sensor_thread()
+    _start_monitor(instance)
     _start_console(instance)
 
 
+def monitor(instance="default"):
+    while True:
```

Comment:
I'm just thinking if it consumes cpu.

A rough idea is:
- Polling bmc pid with a time interval, e.g. 5 seconds, instead of no interval here.
- When user try to connect to SSH server, take a look at bmc pid as well, refuse connection if no bmc and stop ipmi-console at the same time, it can be done with a event.

How do you think?